app/controllers/admin/admin_users_controller.rb

```diff
@@ -87,9 +87,18 @@ def update_next_of_kin
       return
     end
 
+    previous_fnok_user_id = fnok&.kin&.id
     fnok = @user.build_fannish_next_of_kin if fnok.blank?
     fnok.assign_attributes(kin: kin, kin_email: kin_email)
     if fnok.save
+      if previous_fnok_user_id
+        @user.create_log_item({
```

Comment:
This logic is very similar to the case at line 76, I am wondering if we can simplify the method by refactoring.